pkgs/tools/networking/ndn-tools/default.nix

```
@@ -0,0 +1,53 @@
+{ lib
+, stdenv
+, ndn-cxx
+, openssl
+, doxygen
+, boost
+, sqlite
+, pkgconfig
+, python3
+, python3Packages
+, wafHook
+, libpcap
+, fetchFromGitHub
+}:
+let
+  pname = "ndn-tools";
+  version = "0.7.1";
+in
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "named-data";
+    repo = pname;
+    rev = "${pname}-${version}";
+    sha256 = "1q2d0v8srqjbvigr570qw6ia0d9f88aj26ccyxkzjjwwqdx3y4fy";
+  };
+
+  nativeBuildInputs = [ pkgconfig wafHook doxygen python3 python3Packages.sphinx ];
+  buildInputs = [ ndn-cxx boost libpcap openssl ];
+
+  wafConfigureFlags = [
+    "--boost-includes=${boost.dev}/include"
+    "--boost-libs=${boost.out}/lib"
+    # "--with-tests"
+  ];
+
+  # Upstream's tests don't all pass!
+  doCheck = false;
+  checkPhase = ''
+    LD_LIBRARY_PATH=build/ build/unit-tests
+  '';
+
+  # outputs = [ "dev" "out" ];
+
+  meta = with lib; {
+    homepage = "http://named-data.net/";
+    description = "Named Data Neworking (NDN) Essential Tools";
+    license = licenses.lgpl3;
```

According to the [file header](https://github.com/named-data/ndn-tools/blob/master/core/version.cpp.in) the license is `GPLv3+`.

```suggestion
    license = licenses.gpl3Plus;
```